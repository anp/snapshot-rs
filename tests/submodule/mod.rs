
#[cfg(test)]
mod tests {
    use snapshot::snapshot;

    #[snapshot]
    fn test_if_two() -> u32 {
        2
    }
}