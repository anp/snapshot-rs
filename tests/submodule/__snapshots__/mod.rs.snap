{
  "simple::submodule::tests::test_if_two": {
    "file": "tests/submodule/mod.rs",
    "module_path": "simple::submodule::tests",
    "test_function": "test_if_two",
    "recorded_value": 2
  }
}